File: app/src/main/java/com/starostinvlad/fan/GsonModels/Ordinal.java
package com.starostinvlad.fan.GsonModels;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Ordinal implements Serializable {

    @SerializedName("season")
    @Expose
    private Integer season;
    @SerializedName("episode")
    @Expose
    private Integer episode;

    public Integer getSeason() {
        return season;
    }

    public void setSeason(Integer season) {
        this.season = season;
    }

    public Integer getEpisode() {
        return episode;
    }

    public void setEpisode(Integer episode) {
        this.episode = episode;
    }

}

File: app/src/main/java/com/starostinvlad/fan/GsonModels/Ordinal.kt
package com.starostinvlad.fan.GsonModels

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable
import kotlin.Throws

class Ordinal : Serializable {
    @SerializedName("season")
    @Expose
    var season: Int? = null

    @SerializedName("episode")
    @Expose
    var episode: Int? = null
}